.github/workflow-scripts/generate-params.sh

```diff
@@ -27,8 +30,12 @@ cat .github/modified_files.diff
 echo "::endgroup::"
 
 # Generate test categories
-MATRIX_RESULT=$(java .github/workflow-scripts/GenerateCategories.java .github/pull_request_body.txt)
+echo "::group::GenerateCategories Debug"
+java .github/workflow-scripts/GenerateCategories.java .github/pull_request_body.txt 1>> $OUTPUT_FILE
```

Comment:
I am confused about something.  Please help me understand this correctly.  In `.github/workflow-scripts/GenerateCategories.java` you send debug information to System.err, but here it looks like you are only capturing `stdout` (1>>) to the $OUTPUT_FILE.  Where does the `stderr` output go?

Reply:
stdout will be written to the output file and read into the `MATRIX_RESULT` variable. 
stderr will be written to the console so that it is viewable by the developer 
![Screen Shot 2021-04-20 at 2 53 19 PM](https://user-images.githubusercontent.com/33664635/115456000-452d1880-a1e8-11eb-9bac-b16d12052369.png)